BuildForPublication.cmd

```diff
@@ -21,7 +21,7 @@ setlocal
     choice /T 10 /D N /C YN /M "WARNING: Repository is not clean. Press Y to continue or N to stop. "
```

Comment:
May be worthwhile to change the text slightly to be a bit more clear - something like "WARNING: The repo contains uncommitted changes and you're building for publication."  Something along those lines.